src/partition/CmdLineUtils.cc

```diff
@@ -152,13 +154,25 @@ InputLines const makeInputLines(ConfigStore const& config) {
                 }
             }
         }
+        bIsParquetFile = (boost::algorithm::ends_with(s.c_str(), ".parquet") ||
+                          boost::algorithm::ends_with(s.c_str(), ".parq"));
     }
     if (paths.empty()) {
         throw std::runtime_error(
                 "No non-empty input files found among the "
                 "files and directories specified via --in.path.");
     }
-    return InputLines(paths, blockSize * MiB, false);
+
+    // Arrow : collect parameter name list to be read from parquet file
+    std::vector<std::string> names;
+    if (config.has("out.csv.field")) names = config.get<std::vector<std::string>>("in.csv.field");
```

Comment:
Should the config option names be identical here ("out.csv.field" and "in.csv.field" looks inconsistent)?

Reply:
out.csv.field is replaced by in.csv.field